### PR TITLE
Stickers: Add Human Readable Names

### LINKED ISSUE #7425: Stickers Panel: Add human readable names to template stickers
## Feature Description

Add human readable names to the stickers in the stickers panel for accessibility. Other things to consider:
- May need to add the same name to the layer in the layer panel

## Alternatives Considered

n/a

## Additional Context

Stems from [this comment](https://github.com/google/web-stories-wp/pull/7408#issuecomment-834528420).

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Adds human readable names to stickers. This improves accessibility for them and the layers panel

## Summary
Adds `<title>` element to all sticker svgs for improved accessibility and also improves the layers panel with human readable names for the stickers.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
 Stickers used in the templates should now have a more readable name than `Sticker`
 
Screen Shot 2021-05-11 at 1 24 04 PM

## Testing Instructions
Verify user facing changes shown above in layers panel

### QA
Follow Testing instructions above

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7425

### CODE REVIEW

File: packages/stickers/package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
   "type": "module",
   "dependencies": {
     "prop-types": "^15.7.2",
-    "react": "^17.0.2"
+    "react": "^17.0.2",
+    "@web-stories-wp/i18n": "*"
```
Comment (BrittanyIRL) [inline packages/stickers/package.json]: Was this intentional?

Comment (maxyinger) [reply packages/stickers/package.json]: yea, it needs this dep to have access to `__(...)` in the stickers for their titles

File: docs/external-template-creation.md [hunk 2]
```diff
@@ -32,9 +32,10 @@ To add a new template to the editor:
 
 To add new stickers to the codebase, obtain the raw svg file, and view it in your code editor. Paste the contents of the raw svg file into a react component in `packages/stickers/src/<sticker_name>.js`.
 
-Remove extraneous attributes on the base `svg` component and make sure your component takes a `style` property that it applies to the base svg element. Also make sure to remove any explicit `height` and `width` attributes and see that they only are applied to the viewbox. By the end, your component should look something like this:
+Remove extraneous attributes on the base `svg` component and make sure your component takes a `style` property that it applies to the base svg element. Also make sure to remove any explicit `height` and `width` attributes and see that they only are applied to the viewbox. Lastly, be sure to add a title for accessibility. By the end, your component should look something like this:
```
Comment (samwhale) [inline docs/external-template-creation.md:35]: 👍

**Review by samwhale — APPROVED** [review]
Noice

File: packages/stickers/src/beauty/cta.js [hunk 3]
```diff
@@ -16,8 +16,11 @@
 /**
  * External dependencies
  */
+import { __ } from '@web-stories-wp/i18n';
 import PropTypes from 'prop-types';
 
+const title = __('Heart', 'web-stories');
```
Comment (swissspidy) [inline packages/stickers/src/beauty/cta.js]: Let's use `_x()` everywhere so translators have more context.

```suggestion
const title = _x('Heart', 'sticker name', 'web-stories');
```

**Review by swissspidy — CHANGES_REQUESTED** [review]
Strings should use `_x()` to have context.

**Comment by swissspidy** [comment]
There seem to be some unwanted changes to the lock file, breaking webpack builds.

**Review by embarks — APPROVED** [review]

**Review by spacedmonkey — APPROVED** [review]

**Review by swissspidy — APPROVED** [review]

**Pull request merged**